pkgs/by-name/lu/lurk/package.nix

```
@@ -22,16 +22,19 @@ rustPlatform.buildRustPackage rec {
       --replace-fail '/usr/bin/ls' 'ls'
   '';
 
-  meta = with lib; {
+  meta = {
+    changelog = "https://github.com/jakwai01/lurk/releases/tag/${src.rev}";
```

```suggestion
    changelog = "https://github.com/jakwai01/lurk/releases/tag/v${version}";
```
Moving to `tag` unset `rev`. You can re-use version here or `src.tag` again, up to you

---

Yeah, just got notified of the eval failure, I was surprised it wasn't using version before, thanks anyway!